vignettes/mlr3spatiotempcv.Rmd

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Get Started"
-output: rmarkdown::html_vignette
+output: bookdown::html_vignette2
```

Comment:
not sure, if you like this. I have used *bookdown*  to enable numbered figure captions.

Reply:
The vignette will move into the book soon after the CRAN release anyways but for now I don't care too much :)